dashboard/test/ui/features/hour_of_code/tutorial_landing_pages_3.feature

```diff
@@ -0,0 +1,18 @@
+@eyes
+@single_session
+Feature: Looking at tutorial landing pages on Pegasus part Three
+
+Scenario Outline: Simple page view
+  When I open my eyes to test "<test_name>"
+  And I am on "<url>"
+  And I dismiss the language selector
+  And I wait for the video thumbnails to load
+  Then I see no difference for "initial load"
+  And I close my eyes
+  And I sign out
+Examples:
+  | url                                                               | test_name                  |
```

Comment:
Are these the headers of a table? Or leftover from something?

Reply:
These are the headers of the table columns that are used in the lines above to insert the name and url of each line:)